cpp/dolfinx/mesh/Geometry.h

```diff
@@ -117,10 +174,29 @@ class Geometry
   /// (num_points, 3)
   std::span<value_type> x() { return _x; }
 
-  /// @brief The elements that describes the geometry maps.
+  /// @brief The element that describes the geometry map.
   ///
   /// @return The coordinate/geometry element
-  const fem::CoordinateElement<value_type>& cmap() const { return _cmap; }
+  const fem::CoordinateElement<value_type>& cmap() const
+  {
+    if (_cmaps.size() != 1)
+      throw std::runtime_error("Multiple cmaps");
+    return _cmaps[0];
+  }
+
+  /// @brief The element that describe the `i`th geometry map
```

Comment:
I don't follow what this is supposed to do. The docs say it returns the `i`th cmap but it actually returns a list of all of them.

Reply:
Yes, that is wrong. I guess I was in two minds whether just to return a reference to the vector, or access individual items, and it should be the same for `cmaps` and `dofmaps`...